src/gui_impl/signal_browser/signal_graphics_item.cpp

```diff
@@ -457,8 +457,8 @@ void SignalGraphicsItem::mousePressEvent (QGraphicsSceneMouseEvent * event )
                     //check whether a user added stream has already been existing
                     XDFdata->userAddedStream = XDFdata->streams.size();
                     XDFdata->streams.emplace_back();
-                    std::time_t currentTime = std::time(nullptr);
-                    std::string timeString = std::asctime(std::localtime(&currentTime));
+                    time_t currentTime = time(nullptr);
```

Comment:
I was wondering if this change is necessary and why? Anyway, thanks for your patches, I'll review them soon and if everything works fine I'll merge.